shuup/core/suppliers/base.py

```diff
@@ -7,11 +7,21 @@
 # LICENSE file in the root directory of this source tree.
 from django.core.exceptions import ValidationError
 from django.utils.translation import ugettext_lazy as _
+from enumfields import Enum
 
 from shuup.core.models import StockBehavior
 from shuup.core.stocks import ProductStockStatus
 
 
+class StockAdjustmentType(Enum):
```

Comment:
You could probably move this under suppliers/enums.py or something like that to dodge import problems.